CHANGES

```diff
@@ -20,6 +23,7 @@ Features added
 * Add :event:`config-inited` event
 * Add ``sphinx.config.Any`` to represent the config value accepts any type of
   value
+* Add :confval:`author` as a configuration value
```

Comment:
So far, `author` is defined as local variable in `conf.py`.
This enables all of Sphinx modules can refer it.